gradle.properties

```
@@ -111,12 +111,18 @@ artifactRedirecting.androidx.collection.version=1.4.0
 artifactRedirecting.androidx.annotation.version=1.8.0-alpha02
 # TODO: per-target-versioning for annotation is temporary, delete it and keep the line above (after stable 1.8.0)
 artifactRedirecting.androidx.annotation.targetVersions=jvm=1.7.1,default=1.8.0-alpha02
-artifactRedirecting.androidx.lifecycle.version=2.8.0-alpha02
+artifactRedirecting.androidx.lifecycle.version=2.8.0-alpha03
+artifactRedirecting.androidx.navigation.version=2.8.0-alpha05
+artifactRedirecting.androidx.savedstate.version=1.2.1
 
-# Here we define the versions of the libraries published by JB
+# Versions of the libraries published by JetBrains
+# TODO Use non-COMPOSE versions for ANNOTATION and COLLECTION
+jetbrains.publication.version.CORE_BUNDLE=1.0.0-SNAPSHOT
 # Note: an environment variable COMPOSE_CUSTOM_VERSION has a higher priority than the gradle property
-jetbrains.publication.version.COMPOSE=0.1.0-SNAPSHOT
-jetbrains.publication.version.LIFECYCLE=0.1.0-SNAPSHOT
+jetbrains.publication.version.COMPOSE=1.6.10-SNAPSHOT
```

```suggestion
jetbrains.publication.version.COMPOSE=0.1.0-SNAPSHOT
jetbrains.publication.version.LIFECYCLE=0.1.0-SNAPSHOT
jetbrains.publication.version.NAVIGATION=0.1.0-SNAPSHOT
jetbrains.publication.version.SAVEDSTATE=0.1.0-SNAPSHOT
```
Otherwise we have 2 issues:
- 2 source of truth (CI and this)
- if we mistakenly didn't override version on CI, we accidentally publish a wrong version that isn't `0.*.*`

---

Let's have it 0.0.0 here then. In future I'd remove it from here entirely and reset to 0.0.0 during reading